src/helpers/channels/get_stage_instance.ts

```diff
@@ -0,0 +1,21 @@
+import { cacheHandlers } from "../../cache.ts";
+import { rest } from "../../rest/rest.ts";
+import { ChannelTypes } from "../../types/channels/channel_types.ts";
+import { StageInstance } from "../../types/channels/stage_instance.ts";
+import { Errors } from "../../types/discordeno/errors.ts";
+import { endpoints } from "../../util/constants.ts";
+
+/** Gets the stage instance associated with the Stage channel, if it exists. */
+export async function getStageInstance(channelId: bigint) {
+  const channel = await cacheHandlers.get("channels", channelId);
+  if (!channel) throw new Error(Errors.CHANNEL_NOT_FOUND);
```

Comment:
Do not throw an error if the channel is not cached instead use `if (channel) // do checks`